### PR TITLE
Hierarchical Event Descriptors (HED) annotations update

### PR DESCRIPTION
**Pull request by tpatpa**
Update the BIDS starter kit annotations page and template files with respect to HED schemas.

### CODE REVIEW

**Comment by Remi-Gau** [comment]
@tpatpa let me know when this is ready and you want me to review

**Comment by tpatpa** [comment]
Thanks @Remi-Gau !
Where can I find more information on the Netlify checks that failed?

**Comment by Remi-Gau** [comment]
don't worry about those for now: this is an infrastructure related issue that I have to fix. 
not your fault

**Comment by tpatpa** [comment]
> don't worry about those for now: this is an infrastructure related issue that I have to fix. not your fault

Got it, thanks!

@VisLab , @dorahermes could you please take a look at the proposed changes?

File: src/tutorials/annotation.md [hunk 1]
```diff
@@ -108,14 +135,20 @@ and links to helpful guides and tutorials with the details.
...
 
-Ideally, most of this information should be in a single `events.json` sidecar file located in the root directory of your dataset, where it is easy to find and update.
+Ideally, most of this information should be in a single `events.json` sidecar file
+located in the root directory of your dataset,
+where it is easy to find and update.
 
-An overview of how event annotation works in BIDS, as well as tutorials about using available online tools to facilitate annotation, can be found in the
+An overview of how event annotation works in BIDS as well as tutorials
+about using available online tools to facilitate annotation can be found in the
 [BIDS annotation quickstart](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html).
```
Comment (VisLab) [inline src/tutorials/annotation.md:147]: A better link is: 
https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html

File: src/tutorials/annotation.md [hunk 2]
```diff
@@ -128,7 +161,8 @@ annotations in Excel or another tool.
 Instructions for doing this are available in the
 [Spreadsheet templates tutorial](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates).
```
Comment (VisLab) [inline src/tutorials/annotation.md:162]: A better link is:

https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html#spreadsheet-templates

File: src/tutorials/annotation.md [hunk 3]
```diff
@@ -128,7 +161,8 @@ annotations in Excel or another tool.
...
 
-This process and templates make it convenient to provide basic descriptions, as well as HED tags for your dataset events.
+This process and templates make it convenient to provide basic
+descriptions, as well as HED tags for your dataset events.
 
 A [HED annotation quickstart](https://hed-examples.readthedocs.io/en/latest/HedAnnotationQuickstart.html)
```
Comment (VisLab) [inline src/tutorials/annotation.md:167]: A better link is:
https://www.hed-resources.org/en/latest/HedAnnotationQuickstart.html

**Review by VisLab — COMMENTED** [review]
Suggest updating the links.....Otherwise, fine.

File: src/tutorials/annotation.md (hunk 2 above)
Comment (tpatpa) [reply src/tutorials/annotation.md]: Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).

File: src/tutorials/annotation.md (hunk 3 above)
Comment (tpatpa) [reply src/tutorials/annotation.md]: Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).

File: src/tutorials/annotation.md (hunk 1 above)
Comment (tpatpa) [reply src/tutorials/annotation.md]: Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).

**Comment by tpatpa** [comment]
> Suggest updating the links.....Otherwise, fine.

Thank you @VisLab !
Links were updated in the latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).

**Review by VisLab — APPROVED** [review]
One minor grammar correction otherwise good.

File: src/tutorials/annotation.md [hunk 4]
```diff
@@ -130,34 +135,47 @@ and links to helpful guides and tutorials with the details.
...
+
+#### HED schemas
+
+The HED tags used to annotate data come from a controlled vocabulary called a HED schema. A HED schema is a structured vocabulary of terms consisting of top-level tags representing general categories in this vocabulary. Each top-level tag is the root of a tree containing tags falling into that category. This structure allows detailed and accurate annotation of events, machine validation of the annotations, and event description-based search across data collected in various studies.
+
+The rules for HED schema vocabularies and HED-compliant tools can be found in the [HED Specification](https://hed-specification.readthedocs.io/en/latest/).
+
+#### HED library schemas
+
+The HED standard schema contains basic terms that are common across most human neuroimaging, behavioral, and physiological experiments. The HED ecosystem schema libraries extend the standard HED schema with structured vocabulary, including terms unique to specific research fields. This allows the expansion of the HED vocabulary in a scalable manner to support specialized data annotations, for instance, electrophysiological events or language stimuli.
```
[the remark this answers is not in the document]
Comment (tpatpa) [reply src/tutorials/annotation.md]: Corrected, thank you.

**Review by VisLab — APPROVED** [review]

File: src/tutorials/annotation.md [hunk 5]
```diff
@@ -130,34 +135,47 @@ and links to helpful guides and tutorials with the details.
...
-descriptions as well as HED tags for your dataset events.
+descriptions, as well as HED tags for your dataset events.
 
-A [HED annotation quickstart](https://hed-examples.readthedocs.io/en/latest/HedAnnotationQuickstart.html)
+A [HED annotation quickstart](https://www.hed-resources.org/en/latest/HedAnnotationQuickstart.html)
 outlines a step-by-step process for selecting HED tags during the annotation process.
+
+#### HED schemas
+
+The HED tags used to annotate data come from a controlled vocabulary called a HED schema. A HED schema is a structured vocabulary of terms consisting of top-level tags representing general categories in this vocabulary. Each top-level tag is the root of a tree containing tags falling into that category. This structure allows detailed and accurate annotation of events, machine validation of the annotations, and event description-based search across data collected in various studies.
```
Comment (Remi-Gau) [inline src/tutorials/annotation.md:172]:
```suggestion
The HED tags used to annotate data come from a controlled vocabulary called a HED schema.
A HED schema is a structured vocabulary of terms consisting of top-level tags representing general categories in this vocabulary.
Each top-level tag is the root of a tree containing tags falling into that category.
This structure allows detailed and accurate annotation of events,
machine validation of the annotations,
and event description-based search across data collected in various studies.
```

File: src/tutorials/annotation.md [hunk 6]
```diff
@@ -130,34 +135,47 @@ and links to helpful guides and tutorials with the details.
...
+
+The rules for HED schema vocabularies and HED-compliant tools can be found in the [HED Specification](https://hed-specification.readthedocs.io/en/latest/).
+
+#### HED library schemas
+
+The HED standard schema contains basic terms that are common across most human neuroimaging, behavioral, and physiological experiments. The HED ecosystem schema libraries extend the standard HED schema with structured vocabularies, including terms unique to specific research fields. This allows the expansion of the HED vocabulary in a scalable manner to support specialized data annotations, for instance, electrophysiological events or language stimuli.
```
Comment (Remi-Gau) [inline src/tutorials/annotation.md:178]:
```suggestion
The HED standard schema contains basic terms that are common across most human neuroimaging, behavioral, and physiological experiments.
The HED ecosystem schema libraries extend the standard HED schema with structured vocabularies, including terms unique to specific research fields.
This allows the expansion of the HED vocabulary in a scalable manner to support specialized data annotations,
for instance, electrophysiological events or language stimuli.
```

File: src/tutorials/annotation.md [hunk 7]
```diff
@@ -130,34 +135,47 @@ and links to helpful guides and tutorials with the details.
...
+
+Additional details about particular schemas can be found on the [HED schemas](https://hed-schemas.readthedocs.io/en/latest/index.html) documentation page.
+See [HED schema developer’s guide](https://www.hed-resources.org/en/latest/HedSchemaDevelopersGuide.html) to begin developing your own library schema or contribute to existing HED vocabularies.
```
Comment (Remi-Gau) [inline src/tutorials/annotation.md:181]:
```suggestion
See [HED schema developer’s guide](https://www.hed-resources.org/en/latest/HedSchemaDevelopersGuide.html)
to begin developing your own library schema or contribute to existing HED vocabularies.
```

**Review by Remi-Gau — APPROVED** [review]

**Comment by Remi-Gau** [comment]
@tpatpa if you got nothing else to change, am happy to merge this.

FYI I added some semantic line breaks: https://sembr.org/

Content was not changed

**Review by VisLab — APPROVED** [review]

File: src/tutorials/annotation.md [hunk 8]
```diff
@@ -1,17 +1,21 @@
...
 
-Annotation refers to metadata that is directly associated with  data.
+Annotation refers to metadata that is directly associated with data.
 Without adequate annotation, your valuable shared data may be of limited
 use to other researchers and to you in the future.
 
 While the BIDS requirements for annotation are limited,
 BIDS supports a framework for inserting comprehensive
 data annotation at several levels in the dataset.
+In addition, BIDS supports annotations of events and subject characteristics using
```
Comment (dorahermes) [inline src/tutorials/annotation.md:12]: I would say 'For example' rather than 'In addition'

File: src/tutorials/annotation.md [hunk 9]
```diff
@@ -130,34 +135,56 @@ and links to helpful guides and tutorials with the details.
...
+and event description-based search across data collected in various studies.
+
+The rules for HED schema vocabularies and HED-compliant tools can be found in the [HED Specification](https://hed-specification.readthedocs.io/en/latest/).
+
+#### HED library schemas
+
+The HED standard schema contains basic terms that are common across most human neuroimaging, behavioral, and physiological experiments.
+The HED ecosystem schema libraries extend the standard HED schema with structured vocabularies, including terms unique to specific research fields.
+This allows the expansion of the HED vocabulary in a scalable manner to support specialized data annotations,
+for instance, electrophysiological events or language stimuli.
```
Comment (dorahermes) [inline src/tutorials/annotation.md:186]: Add links to the words of electrophysiological events (to HED-SCORE) and language stimuli (to LISA)

**Review by dorahermes — COMMENTED** [review]
Juts some small suggestions

File: src/tutorials/annotation.md (hunk 8 above)
Comment (tpatpa) [reply src/tutorials/annotation.md]: Changed [here](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/fb1b7d87083388d5969afa7c10e59cca0b88214c)

File: src/tutorials/annotation.md (hunk 9 above)
Comment (tpatpa) [reply src/tutorials/annotation.md]: Added [here](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/fb1b7d87083388d5969afa7c10e59cca0b88214c)

**Comment by Remi-Gau** [comment]
@tpatpa let me know when this is good to go on your side

**Comment by tpatpa** [comment]
Thanks @Remi-Gau !
I have addressed reviewers comments so we can go ahead and merge.

**Comment by Remi-Gau** [comment]
merging then

**Pull request merged**